doc/api/esm.md

```diff
@@ -1064,11 +1064,17 @@ and parent URL. The module specifier is the string in an `import` statement or
 `import()` expression, and the parent URL is the URL of the module that imported
 this one, or `undefined` if this is the main entry point for the application.
 
+The `conditions` property on the `context` is an array of conditions for
+[Conditional Exports][] that apply to this resolution request. They can be used
+for looking up conditional mappings elsewhere or to extend the list when calling
+the default resolution logic.
```

Comment:
Thanks for adding this. @nodejs/modules-active-members does this seem understandable and good UX to you?

Reply:
makes sense to me